integration-tests/subscriptions.test.ts

```diff
@@ -86,6 +86,90 @@ if (SUBSCRIPTIONS_ENABLED === 'true') {
                 expect(notifications).toEqual([]);
             });
         }
+
+        test('end to end test with id notifications', async () => {
+            const uuid = v4();
+            const subResource = randomSubscription(uuid);
+            // 1. Create a Subscription.
+            const postSubscriptionResult = await client.post('Subscription', subResource);
+            expect(postSubscriptionResult.status).toBe(201);
+            // 2. Create/Update a resource that matches the subscription.
+            const postPatientResult = await client.post('Patient', resourceThatMatchesSubscription);
+            expect(postPatientResult.status).toEqual(201);
+            // 3. Verify that notifications are received
+            // give 2 minutes for notification to be placed in ddb table
+            await new Promise((r) => setTimeout(r, 120_000));
```

Comment:
This is used several times, It'd be better to declare this helper function once. 
You can move an existing `sleep` fn to test utils: https://github.com/awslabs/fhir-works-on-aws-deployment/blob/f122394c3f06ff380b51cbb513dc92f83a49347c/integration-tests/bulkExport.test.ts#L12